config.js

```diff
@@ -375,6 +375,20 @@ var config = {
         // shard: "shard1",
         // region: "europe",
         // userRegion: "asia"
+    },
+
+    // Information for the chrome extension banner
+    chromeExtensionBanner: {
+        // The chrome extension to be installed address
+        url: 'https://chrome.google.com/webstore/detail/jitsi-meetings/kglhbbefdnlheedjiejgomgmfplipfeb',
+
+        // Extensions info which allows checking if they are installed or not
+        chromeExtensionsInfo: [
+            {
+                id: 'kglhbbefdnlheedjiejgomgmfplipfeb',
```

Comment:
Same here.

Reply:
if the chromeExtensionInfo is missing the component will not render. Also now it shows a warning in console as well.